api/_lib/parser.ts

```diff
@@ -59,14 +59,27 @@ const textImageRequestParser = (
 const featuredImageRequestParser = (
   encodedQuery: string
 ): FeaturedImageRequest => {
-  const { imageURL, bg } = JSON.parse(
+  const { imageURL, bg, scale, borderRadius } = JSON.parse(
     Buffer.from(encodedQuery, "base64").toString("binary")
   );
```

Comment:
Can `JSON.parse` throw an error?

Reply:
Yes but there is a try-catch here https://github.com/makelogio/og-image/blob/e61dbb06061ff409040350e9c40609a9a31426ff/api/index.ts#L33